verilog/tools/syntax/export_json_examples/BUILD

```diff
@@ -0,0 +1,21 @@
+load("@rules_python//python:defs.bzl", "py_test", "py_library")
+load("@pip//:requirements.bzl", "requirement")
```

Comment:
Using requirements is essentially side-stepping the idea of an hermetic build provided by bazel.

We should see where we can get 'anytree' and 'dataclasses' defined as a dependency and then provide these explicitly in the `deps = []` in the `verible_verilog_syntax_py` library. I have [seen projects](https://github.com/bazelbuild/bazel/blob/master/third_party/BUILD) use something like "//third_party/py/dataclasses" dependencies (but there, they build it from source somewhere. So I suspect we'll end up with something @python_dataclasss and @python_anytree  in the WORKSPACE file.)

How this needs to be integrated here I'll in detail, I don't know, so I have to defer to you to find out :)

Reply:
I've used two ways of handling third party dependencies:
* anytree is installed through WORKSPACE, with dedicated BUILD file. This is how other external dependencies are handled in Verible (i.e. flex, bison, m4, and now also jsoncpp and anytree)
* dataclasses (BUILD file and sources) is located in third_party. The backport from pypi doesn't work with Python 3.7+, so I've copied the backport from Bazel sources.

Related commits:
https://github.com/google/verible/pull/616/commits/709c06e05d056387f9ef22925eecc829af1fbfdf
https://github.com/google/verible/pull/616/commits/baab531628c3e4334872c87ff56d7106e60309ac